data/net/minecraft/world/inventory/Slot.mapping

```diff
@@ -1,4 +1,9 @@
 CLASS net/minecraft/world/inventory/Slot
+	METHOD <init> (Lnet/minecraft/world/Container;III)V
+		ARG 1 container
+		ARG 2 containerIndex
```

Comment:
Point of 'parameters follow field names': `containerIndex` -> `slot`. Subclasses should be modified to follow this too (at least, those within scope of this PR.)